src/db_tables/_media.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import ForeignKey
+from sqlalchemy.orm import Mapped, mapped_column
+
+# Prevent circular imports for relationships
+
+
+class Media:
+    """Represents a media object in the database. DO NOT USE DIRECTLY."""
+
+    id: Mapped[int] = mapped_column(primary_key=True, index=True, nullable=False)
```

Comment:
It would be nice if meda had a `tvdb_id`, and the primary key was a composite of that and the user id

Reply:
I don't have to change anyhting tho, apart to make this a str?

Reply:
See https://docs.sqlalchemy.org/en/20/orm/composites.html#mapper-composite and https://docs.sqlalchemy.org/en/20/orm/composites.html#composite-api

Reply:
k thanks